src/test/java/core/basesyntax/validation/UserValidatorImplTest.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax.validation;
+
+import core.basesyntax.exception.UserValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class UserValidatorImplTest {
+    private final UserValidator validator = new UserValidatorImpl();
+
+    @Test
+    public void validate_ValidUser_ok() {
+        User user = new User();
+        user.setAge(18);
+        user.setLogin("validated");
+        user.setPassword("validated");
+        assertDoesNotThrow(() -> validator.validate(user));
+    }
+
+    @Test
+    public void validate_UserIsNull_notOk() {
+        User user = null;
+        assertThrows(UserValidationException.class, () -> validator.validate(user));
+    }
+
+    @Test
+    public void validate_LoginIsNull_notOk() {
+        User user = new User();
+        user.setLogin(null);
+        assertThrows(UserValidationException.class, () -> validator.validate(user));
+    }
+
+    @Test
+    public void validate_LoginIsLessThan6Characters_notOk() {
+        User user = new User();
+        user.setLogin("null");
+        assertThrows(UserValidationException.class, () -> validator.validate(user));
```

Comment:
The test name `validate_LoginIsLessThan6Characters_notOk` suggests that you are testing for a login with less than 6 characters, but the login 'null' actually has 4 characters. Consider adding a test case for a login with exactly 5 characters to properly test the edge case.